numerics_paper/version2/numerical_chj.tex

```diff
@@ -473,12 +484,19 @@ \section{An entropy-based blending of Rusanov and Roe}\label{sec:blended_rs}
 
 \subsection{The entropy residual}
 Let $\entvar(\bfq)$ and $\entflux(\bfq)$ denote the entropy variable and entropy flux.
+Recall that $Q=[h,hu,hv]^t$. We use the total energy as entropy; that is, we consider 
+\begin{align*}
+  \eta(\bfq)=\frac{1}{2} gh^2 + \frac{(hu)^2+(hv)^2}{2h}, \qquad
```

Comment:
Again, I think we should just have "q" here.

Reply:
Agree